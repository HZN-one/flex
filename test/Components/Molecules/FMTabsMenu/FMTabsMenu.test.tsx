/**
 * @jest-environment jsdom
 */
import * as React from "react";
import "@testing-library/jest-dom";
import { FMTabsMenu } from "@Molecules";
import { render } from "@testing-library/react";

describe("FMStepper", () => {
  it("should match snapshot", () => {
    const { container } = render(
      <FMTabsMenu
        testID="tabs-example"
        tabs={[
          { label: "Individual", iconName: "person_outline" },
          { label: "Company", iconName: "home_work" },
        ]}
      />
    );
    expect(container).toMatchSnapshot();
  });
});
