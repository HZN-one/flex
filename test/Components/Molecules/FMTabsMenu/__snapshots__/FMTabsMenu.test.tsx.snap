// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FMStepper should match snapshot 1`] = `
<div>
  <div
    class="TabsUnstyled-root TabsUnstyled-vertical css-1j1tgpi"
    data-testid="tabs-example"
  >
    <div
      aria-orientation="vertical"
      class="css-19ihvrq TabsListUnstyled-root TabsListUnstyled-vertical"
      role="tablist"
    >
      <button
        aria-controls="mui-1-P-0"
        aria-selected="true"
        class="TabUnstyled-root Mui-selected css-c8w0k6"
        id="mui-1-T-0"
        role="tab"
        tabindex="0"
      >
        <div
          class="MuiBox-root css-5i61f4"
        >
          <span
            aria-hidden="true"
            class="material-icons notranslate MuiIcon-root MuiIcon-fontSizeMedium css-1a42aau-MuiIcon-root"
            data-testid="icon-tab"
          >
            person_outline
          </span>
          <p
            class="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
            data-testid="typography-tab"
          >
            Individual
          </p>
        </div>
      </button>
      <button
        aria-controls="mui-1-P-1"
        aria-selected="false"
        class="TabUnstyled-root css-c8w0k6"
        id="mui-1-T-1"
        role="tab"
        tabindex="-1"
      >
        <div
          class="MuiBox-root css-5i61f4"
        >
          <span
            aria-hidden="true"
            class="material-icons notranslate MuiIcon-root MuiIcon-fontSizeMedium css-1a42aau-MuiIcon-root"
            data-testid="icon-tab"
          >
            home_work
          </span>
          <p
            class="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
            data-testid="typography-tab"
          >
            Company
          </p>
        </div>
      </button>
    </div>
  </div>
</div>
`;
